### DISCUSSION TITLE
Raspberry Pi 3A Case

### DISCUSSION THREAD

**Discussion by rcarmo** [probonopd/MiniDexed#601] [Show and tell]
After a long hiatus, I finally published the STLs and OpenSCAD source for [my Raspberry Pi 3A case](https://github.com/rcarmo/openscad-minidexed-rpi3a), which manages to pack in a DAC and a MIDI IN TRS jack:
![box](https://github.com/probonopd/MiniDexed/assets/392683/b4ccc072-5f8e-4316-bed9-642fdafc2c40)

I don't intend to mess about with the design any further, but it may be useful for someone.

**Comment by diyelectromusic** [comment]
That looks great! :)
